src/helpers/getAllCurrencies.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+export const getAllCurrencies = async (setAllCurrencies: any) => {
+    return await axios.get('https://openexchangerates.org/api/currencies.json')
```

Comment:
You should write pure functions. getAllCurrencies should works only with getting data from api.
Read about Pure Functions